File: reference/javastudy/day0106/src/javaStudy/Study01.java
package javaStudy;

public class Study01 {
    public static void main(String[] args) {
        
    /*  println();
        print();
        printf(); */
    
        System.out.println("Java"); // ln = line
        System.out.println("Study"); // ln = line

        System.out.print("Java"); // 줄바꿈 없음
        System.out.print("Study"); // 줄바꿈 없음

        System.out.println(); // 줄바꿈

        int value = 1000;
        System.out.printf("가격 : %d원\n", value); // %d : 10진 정수 형태로 출력
        System.out.printf("가격 : %6d원\n", value); // 6칸 확보 후 오른쪽 정렬
        System.out.printf("가격 : %-6d원\n", value); // 6칸 확보 후 왼쪽 정렬
        System.out.printf("가격 : %06d원\n", value); // 6칸 확보 후 빈칸 0으로 채우기

        double area = 3.14159 * 10 * 10;
        System.out.printf("반지름이 %d인 원의 넓이 : %10.2f\n", 10, area); // 전체 10칸, 소수점 이하 2자리

        String name = "이재명";
        String job = "대통령";
        System.out.printf("%6d | %-10s | %10s", 1, name, job);
    }
}

File: reference/javastudy/day0106/bin/pythonStudy/Study01.py
value = 1000

print(f"가격 : {value}원"); #기본 출력
print(f"가격 : {value:6d}원"); # 전체 자리수 6자리 확보, 오른쪽 정렬
print(f"가격 : {value:<6d}원"); # 전체 자리수 6자리 확보, 왼쪽 정렬
print(f"가격 : {value:06d}원"); # 전체 자리수 6자리 확보, 빈자리 0으로 채우기

area = 3.14159265358979 * 10 * 10
r = 10
print("반지름이 {r}인 원의 넓이: {area:10.2f}"); 

name = "이재명"
job = "대통령"
print(f"{1:6d} | {name:<10} | {job:>10}");